### ISSUE TITLE
lip sync using audio 

### ISSUE THREAD

**Issue by prajwal3232** [KwaiVGI/LivePortrait#156]
is it possible to use this for audio driven lip sync as well motion transfer???

**Comment by Mrkomiljon** [comment]
@prajwal3232 yes, Metahuman + speech graphics + webcam_live_portrait. please check in [my case](https://github.com/Mrkomiljon/Webcam_Live_Portrait)

**Comment by prajwal3232** [comment]
> @prajwal3232 yes, Metahuman + speech graphics + webcam_live_portrait. please check in [my case](https://github.com/Mrkomiljon/Webcam_Live_Portrait)

sure will check it out

**Comment by xuyangcao** [comment]
> is it possible to use this for audio driven lip sync as well as motion transfer???

Based on the idea of ​​VASA-1, we use LivePortrait to implement an audio-driven portrait animation method. For details, please refer to: https://jdh-algo.github.io/JoyVASA/.

**Comment by FurkanGozukara** [comment]
> > is it possible to use this for audio driven lip sync as well as motion transfer???
> 
> Based on the idea of ​​VASA-1, we use LivePortrait to implement an audio-driven portrait animation method. For details, please refer to: https://jdh-algo.github.io/JoyVASA/.

will JoyVASA  be added into liveportrait app or it will be a separate app?

it looks great
